packages/tools/pie-icons-vue/test/IconAlcoholLarge.test.js

```diff
@@ -6,7 +6,8 @@ import {
     vi,
 } from 'vitest';
 
-import IconAlcoholLarge from '../icons/IconAlcoholLarge';
+// eslint-disable-next-line import/no-unresolved, import/extensions
+import IconAlcoholLarge from '../esm/IconAlcoholLarge';
```

Comment:
does this mean consumers will also need to update their imports to use esm?

Reply:
I believe not. The package contains only `dist` and `esm`.